core/src/main/java/com/ttbmp/cinehub/core/datamapper/EmployeeDataMapper.java

```diff
@@ -0,0 +1,55 @@
+package com.ttbmp.cinehub.core.datamapper;
+
+import com.ttbmp.cinehub.core.dto.EmployeeDto;
+import com.ttbmp.cinehub.core.dto.ProjectionistDto;
+import com.ttbmp.cinehub.core.dto.UsherDto;
+import com.ttbmp.cinehub.core.entity.Employee;
+import com.ttbmp.cinehub.core.entity.Projectionist;
+import com.ttbmp.cinehub.core.entity.Usher;
+import com.ttbmp.cinehub.core.utilities.DataMapperHelper;
+
+import java.util.List;
+
+/**
+ * @author Massimo Mazzetti
+ */
+
+public class EmployeeDataMapper {
+    private EmployeeDataMapper() {
+    }
+
+    public static EmployeeDto mapToDto(Employee employee) {
+        if (employee instanceof Usher) {
```

Comment:
Use polymorphism instead of if conditions

Reply:
what do you mean?